module-apps/application-meditation/widgets/TimerProperty.cpp

```diff
@@ -162,9 +194,11 @@ void TimerProperty::State::increment() noexcept
 
 void TimerProperty::State::decrement() noexcept
 {
-    auto it =
-        std::upper_bound(std::rbegin(timeArr), std::rend(timeArr), timeInMinutes, [](int a, int b) { return a > b; });
-    if (it == std::rend(timeArr)) {
+    auto it = std::upper_bound(std::rbegin(Constants::Params::meditationDurations),
+                               std::rend(Constants::Params::meditationDurations),
+                               timeInMinutes,
+                               [](int a, int b) { return a > b; });
```

Comment:
`std::greater`?

Reply:
This was not the purpose of the bug, IMO such an implementation can stay. What do you think?

Reply:
I meant just replacing the lambda with `a > b`, but it's not a big issue. Close if you wish